systemverilog/SystemVerilog.g4

```diff
@@ -3288,7 +3288,8 @@ bit_select
 	: ('[' expression ']')*
 	;
 
-select
+// renamed 'select' to 'select_' because of a conflict with the Go language
```

Comment:
I don't think this comment is required because actually there are many symbol conflicts in different runtimes (search by [symbol-conflict](https://github.com/antlr/grammars-v4/issues?q=label%3Asymbol-conflict+) label) and adding `_` suffix is common practice in our repository.